tests/components/__snapshots__/multi_select.spec.js.snap

```diff
@@ -124,24 +124,24 @@ exports[`MultiSelect can remove search 1`] = `
       }
     }
   >
-    <SourceList
+    <Unknown
       calculatedHeight={360}
-      disabled={false}
+      disabled={undefined}
```

Comment:
The fact that a lot of these changed from actual values to undefined doesn't seem right.

Reply:
This is because it became wrapped with HOC and shallow renderer is not testing deep props...

Reply:
Yes but now they are not tested...

Reply:
ShallowRenderer doesn't fit our needs for this particular case anymore.
Maybe we should then use mount or make snapshot from json instead - in order to be able to check all passed values.

Reply:
What about this: I found library skin-deep that generates array of properties for component
this library + snapshot will give us ability to do deep props checking for this case
please see: https://github.com/MaksimDrobchak/react-multi-select/pull/1/files

Reply:
it may be the possible solution
@liorheber @talyak @yuval-rikover @AmirAsaraf  what do you think about adding this library for tests?

Reply:
guys? someone please?